spec/support/uuid_setting.rb

```diff
@@ -1,6 +1,4 @@
 class UUIDSetter
-  include EnvHelpers
```

Comment:
any reason to remove this?

Reply:
@ediogodias fixed. Tests were failing locally for me that was why.